ci-cd-bootstrap/inventory/group_vars/all.yml

```diff
@@ -0,0 +1,49 @@
+---
+openshift_cluster_content:
+- object: projectrequest
+  content:
+  - name: labs-ci-cd
+    file: "{{ inventory_dir }}/../namespaces/labs-ci-cd.yml"
+  - name: labs-dev
+    file: "{{ inventory_dir }}/../namespaces/labs-dev.yml"
+  - name: labs-test
+    file: "{{ inventory_dir }}/../namespaces/labs-test.yml"
+  - name: labs-uat
+    file: "{{ inventory_dir }}/../namespaces/labs-uat.yml"
+- object: policy
```

Comment:
since we're not using any cluster-wide policies, I'd suggest removing this section to avoid confusion.